.github/workflows/main.yml

```diff
@@ -74,21 +74,45 @@ jobs:
           tag_name: ${{ steps.git_tag.outputs.tag }}
           release_name: Release ${{ steps.git_tag.outputs.tag }}
           body: ${{ steps.git_tag.outputs.changes }}
-      - if: github.ref == 'refs/heads/master'
-        name: Prepare chart for publishing
-        id: chart_prepare
+      - name: Slack Notification
+        uses: rtCamp/action-slack-notify@v2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+          SLACK_CHANNEL: github-ci
+          SLACK_COLOR: ${{ job.status }}
+          SLACK_ICON: https://github.com/redkubes.png?size=48
+          SLACK_TITLE: CI run ${{ job.status }}
+          SLACK_USERNAME: RedKubesBot
+        if: always() # failure
+
+  chart-release:
+    if: "contains(github.event.head_commit.message, 'chore(release)')"
+    needs: build-test-push-release
+    runs-on: ubuntu-latest
+    container:
+      image: otomi/tools:v1.4.19
+      options: --user 0
```

Comment:
is it possible with non-root user?

Reply:
no, as the pipeline runner needs root access to mount certain things...